scout/adapter/mongo/case.py

```diff
@@ -222,6 +221,8 @@ def _populate_name_query(self, query, name_query, owner=None, collaborator=None)
         query_field = name_query.split(":")[0]  # example:status
         query_term = name_query[name_query.index(":") + 1 :].strip()
 
+        LOG.info("Name query: %s Field: %s term: %s", name_query, query_field, query_term)
```

Comment:
```suggestion
```

Reply:
Hehe, I put it back again when reading the comments on #3571, but sure.